hazelcast/src/main/java/com/hazelcast/util/FutureUtil.java

```diff
@@ -92,6 +92,23 @@ public void handleException(Throwable throwable) {
         }
     };
 
+    /**
+     * This ExceptionHandler rethrows {@link java.util.concurrent.ExecutionException}s and logs
+     * {@link com.hazelcast.core.MemberLeftException}s to the log.
+     */
+    public static final ExceptionHandler RETHROW_ALL_EXCEPT_MEMBER_LEFT = new ExceptionHandler() {
```

Comment:
This exception handler doesn't seem to have any usages, maybe remove it from the final PR?